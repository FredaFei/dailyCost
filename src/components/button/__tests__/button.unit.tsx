import * as renderer from 'react-test-renderer'
import React from 'react'
import Button from '../button';

describe('calculate', () => {
  it('Button 存在', () => {
    const json = renderer.create(<Button>hello</Button>).toJSON()
    expect(json).toMatchSnapshot()
  })
});

