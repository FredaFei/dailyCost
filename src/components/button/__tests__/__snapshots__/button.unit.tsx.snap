// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calculate Button 存在 1`] = `
<button
  className="m-component-button-wrapper "
  data-role="Button"
  onClick={[Function]}
>
  <span
    className="m-component-button-content"
  >
    hello
  </span>
</button>
`;
